cert-provider/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine
+
+EXPOSE 80
+WORKDIR /usr/src/app
+VOLUME [ "/usr/src/app/certs" ]
+
+RUN apk add --update curl openssl git socat ncurses
+
+RUN git clone https://github.com/Neilpang/acme.sh.git && \
+    cd acme.sh && \
+    ./acme.sh --install  \
```

Comment:
Can you please pin it to a specific, tested, tag or commit?

Reply:
done